aspnetcore/blazor/components/prerender.md

```diff
@@ -17,358 +17,134 @@ uid: blazor/components/prerender
-
-`PrerenderedCounter1.razor`:
+:::moniker range=">= aspnetcore-8.0"
 
-:::code language="razor" source="~/../blazor-samples/8.0/BlazorSample_BlazorWebApp/Components/Pages/PrerenderedCounter1.razor":::
+Prerendering is enabled by default for interactive components.
 
-Run the app and inspect logging from the component. The following is example output.
+Internal navigation for interactive routing doesn't involve requesting new page content from the server. Therefore, prerendering doesn't occur for internal page requests, including for [enhanced navigation](xref:blazor/fundamentals/routing#enhanced-navigation-and-form-handling). For more information, see [Static versus interactive routing](xref:blazor/fundamentals/routing#static-versus-interactive-routing), [Interactive routing and prerendering](xref:blazor/state-management/prerendered-state-persistence#interactive-routing-and-prerendering), and [Enhanced navigation and form handling](xref:blazor/fundamentals/routing#enhanced-navigation-and-form-handling).
 
-> [!NOTE]
-> If the app adopts [interactive routing](xref:blazor/fundamentals/routing#static-versus-interactive-routing) and the page is reached via an internal [enhanced navigation](xref:blazor/fundamentals/routing#enhanced-navigation-and-form-handling), prerendering doesn't occur. Therefore, you must perform a full page reload for the `PrerenderedCounter1` component to see the following output. For more information, see the [Interactive routing and prerendering](#interactive-routing-and-prerendering) section.
+## Disable prerendering
 
-> :::no-loc text="info: BlazorSample.Components.Pages.PrerenderedCounter1[0]":::  
-> :::no-loc text="      currentCount set to 41":::  
-> :::no-loc text="info: BlazorSample.Components.Pages.PrerenderedCounter1[0]":::  
-> :::no-loc text="      currentCount set to 92":::
+<!-- UPDATE 11.0 Tracking ...
 
-The first logged count occurs during prerendering. The count is set again after prerendering when the component is rerendered. There's also a flicker in the UI when the count updates from 41 to 92.
+                 "prerender: false" is ignored in child components
+                 https://github.com/dotnet/aspnetcore/issues/55635
 
-To retain the initial value of the counter during prerendering, Blazor supports persisting state in a prerendered page using the <xref:Microsoft.AspNetCore.Components.PersistentComponentState> service (and for components embedded into pages or views of Razor Pages or MVC apps, the [Persist Component State Tag Helper](xref:mvc/views/tag-helpers/builtin-th/persist-component-state-tag-helper)).
+                 ... for .NET 11 work in the following area. -->
 
-:::moniker range=">= aspnetcore-10.0"
+Disabling prerendering using the following techniques only takes effect for top-level render modes. If a parent component specifies a render mode, the prerendering settings of its children are ignored.
```

Comment:
This is an important detail, but I don't think it should be the first sentence in this section. Can we move it somewhere near the end of the section instead?